src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    email,
+    name,
+  } = user;
```

Comment:
2 attributes can go in 1 link
```suggestion
  const { email, name } = user;
```

Reply:
done